handlers/backend.go

```diff
@@ -246,7 +249,7 @@ func (h backend) count(w http.ResponseWriter, r *http.Request) error {
 		CreatedAt: goatcounter.Now(),
 	}
 
-	_, err := zhttp.Decode(r, &hit)
+	err := formam.NewDecoder(&formam.DecoderOptions{TagName: "json"}).Decode(r.URL.Query(), &hit)
```

Comment:
Reading the `navigator.sendBeacon` docs on MDN, it looks like it can send the data as form data as well. In this case it doesn't matter too much, but I'm mostly thinking about some other future use cases, and using URL parameters for a POST isn't very common, so it would probably be less surprising to use form data?

Reply:
> so it would probably be less surprising to use form data?

Actually I was surprised that the URL parameter of my POST query didn't work :smile: 

Maybe because I know that the count endpoint works with the URL parameters. I thought that it didn't care about the method.

> I'm mostly thinking about some other future use cases

Do you have concrete use-cases ? Otherwise a dedicated endpoint would probably make more sense one the use-cases get defined (since the `/count` endpoint is called very often, it make probably sense to keep it simple and fast)

Reply:
> Do you have concrete use-cases?

Yeah, don't know; can't really think of any. It's just when someone wants to use it for whatever reason it seemed to me that form params would be less surprising.

But a line in the docs would be fine as well, it doesn't really matter.

Reply:
> But a line in the docs would be fine as well

Sounds good!